### DISCUSSION TITLE
Add Config Option to auto handle updated Config files

### DISCUSSION THREAD

**Discussion by divinewind420** [ThioJoe/YT-Spammer-Purge#691] [Ideas]
Suggest adding: config options (ASK/UPDATE/etc) to automatically handle updating updated config files.

**Comment by ThioJoe** [comment]
Hm what is it about the current way it works that you think needs to be changed? The fact that it stops to prompt you when there's a config file update? I've been considering just doing away with the prompt altogether, it was mostly at the beginning when the feature was new and not so well tested.

**Comment by divinewind420** [comment]
I have config file setup for an automatic run, stopping to press enter a minor inconvenience. Some may want it, so I suggested making it an option.  I'm fine with removing it or changing, either way. 

Sent from Yahoo Mail on Android 
 
  On Tue, Feb 22, 2022 at 7:44 AM, ***@***.***> wrote:   


Hm what is it about the current way it works that you think needs to be changed? The fact that it stops to prompt you when there's a config file update? I've been considering just doing away with the prompt altogether, it was mostly at the beginning when the feature was new and not so well tested.

—
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.
